modules/storage-persistent-storage-pvc.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage.adoc
```

Comment:
I don't see this file included in this or any assembly, should it be?

Reply:
It should! I've included it immediately below the PV module.